cmd/security-secretstore-setup/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM golang:1.12-alpine AS builder
+
+ENV GO111MODULE=on
+
+WORKDIR /edgex-go
+
+RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
+
+RUN apk update && apk add make git
+
+COPY go.mod .
+
+RUN go mod download
+
+COPY . .
+
+RUN make cmd/security-secretstore-setup/security-secretstore-setup
+
+FROM alpine:3.10
+
+LABEL license='SPDX-License-Identifier: Apache-2.0' \
+      copyright='Copyright (c) 2019: Dell Technologies, Inc.'
+
+WORKDIR /
+
+COPY --from=builder /edgex-go/cmd/security-secretstore-setup/res/docker/configuration.toml /res/configuration.toml
+
+COPY --from=builder /edgex-go/cmd/security-secretstore-setup/security-secretstore-setup .
+
+ENTRYPOINT ["secretstore-init.sh"]
+
+#ENTRYPOINT ["./security-secretstore-setup","--init=true"]
```

Comment:
BUG!

`cmd/security-secretstore-setup/secretstore-init.sh` was supposed to
```sh
exec ./security-secretstore-setup --init=true
```
at the end.

Reply:
See https://github.com/edgexfoundry/edgex-go/pull/1719/files/aa672477b08f26601702723eda8d51e78e880a30#r319581789

Recommendation is to go back to `FROM scratch` and put ENTRYPOINT back the way it was.

Reply:
@bnevis-i I thought your proposal is to run the script at the end of the dockerfile and all the binaries will be called in the script, so I have this dummy script called at the end. So if we go back to call the binary directly, where will be script be residing?

Reply:
See discussion on `cmd/security-secretstore-setup/secretstore-init.sh`

Reply:
as we discussed from last Friday this is not a bug. This serves as the holder project, so it will not include any implementation. Line of "exec ./security-secretstore-setup --init=true" should be added into script if we decide to adopt script, or added into the docker file if we decide not to use script. in this sense I would prefer to keep it as is now.

Reply:
As we decided to remove the script option during last week's meeting, this falls back to call the binary directly.